src/helpers/firebaseHelpers.js

```diff
@@ -0,0 +1,6 @@
+export const parseData = (querySnapshot) =>
```

Comment:
I see that you have created also `firebaseUtils.js`.... what's the difference between both files?

Reply:
Well, about naming files. I know it's difficult to choose the best name but in this specific case. I suggested creating those files: `firebaseHelpers.js` and `firebaseOperations.js`. Maybe, it's only a small confusion.

- `firebaseOperations.js`: this file could have functions about Firebase like `addProduct`, `readProduct`, `updateProduct`, and `deleteProduct`.
- `firebaseHelpers.js`: it has small or medium functions that help us to "process" Firebase information as parsing data, filtering, getProductsByName,  getProductsByCreatedAt, and so on.

I'm so sorry for the confusion.